backend/db/models/review.js

```diff
@@ -17,6 +17,10 @@ module.exports = (sequelize, DataTypes) => {
         foreignKey:'spotId',
         onDelete:'cascade'
       })
+      Review.belongsTo(models.User,{
+        foreignKey:"userId",
+        onDelete:'cascade'
```

Comment:
Keep in mind that cascade may cause unintended effects if used here

Reply:
will check that today